packages/admin-next/dashboard/src/routes/pricing/common/utils.ts

```diff
@@ -25,25 +25,30 @@ export const getPriceListStatus = (
 
   let text = t("pricing.status.active")
   let color: "red" | "grey" | "orange" | "green" = "green"
-
-  if (isScheduled) {
-    color = "orange"
-    text = t("pricing.status.scheduled")
-  }
+  let status: string = PriceListStatus.ACTIVE
 
   if (isDraft) {
     color = "grey"
     text = t("pricing.status.draft")
+    status = PriceListStatus.DRAFT
```

Comment:
Why are we using 2 different enums here for draft and the other two states?

Reply:
One is actual db status, the other is computed status that we display on the UI based on the date

Reply:
@riqwan I see, ideally the UI then defines all statuses, basically a union of the computed + db in one enum

Reply:
Hm, I don't know if that adds much value here for a simple badge. If we use it in other places, sure but that doesn't seem to be the case yet.